alerter/alerter.cs

```diff
@@ -1,29 +1,45 @@
 ﻿using System;
+using System.Diagnostics;
 
-namespace AlerterSpace {
-    class Alerter {
+namespace AlerterSpace 
+{
+    class Alerter 
+    {
         static int alertFailureCount = 0;
-        static int networkAlertStub(float celcius) {
-            Console.WriteLine("ALERT: Temperature is {0} celcius", celcius);
+        static int networkAlertStub(float celcius) 
+        {            
             // Return 200 for ok
             // Return 500 for not-ok
-            // stub always succeeds and returns 200
-            return 200;
+            // stub always succeeds and returns 200  
+            if (celcius <= 200)
+            {
+                return 200;
+            }
+            else
+            {
+                Console.WriteLine("ALERT: Temperature is {0} celcius", celcius);
+                return 500;
+            }                 
         }
-        static void alertInCelcius(float farenheit) {
+        static void alertInCelcius(float farenheit) 
+        {
             float celcius = (farenheit - 32) * 5 / 9;
             int returnCode = networkAlertStub(celcius);

```

Comment:
The production code should not be aware of test code or test stub. It should be abstracted by the use of interface or function pointer.